ui/widgets/nuxeo-html-editor.js

```diff
@@ -39,8 +40,76 @@ import './quill/quill-snow.js';
           div#editor > * {
             margin-top: 0;
           }
+
+          iron-icon {
+            height: 18px;
+            color: #444;
+          }
         </style>
 
+        <nuxeo-document-picker
+          id="picker"
+          provider="picker"
+          page-size="40"
+          schemas="dublincore,file"
+          enrichers="thumbnail,permissions,highlight"
+          search-name="picker"
+          on-inserted="_onPickerInserted"
+        ></nuxeo-document-picker>
+
+        <div id="toolbar">
+          <span class="ql-formats">
+            <select class="ql-header">
+              <option value="1"></option>
+              <option value="2"></option>
+              <option value="3"></option>
+              <option value="4"></option>
+              <option value="5"></option>
+              <option value="6"></option>
+              <option selected></option>
+            </select>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-bold"></button>
+            <button class="ql-italic"></button>
+            <button class="ql-underline"></button>
+            <button class="ql-strike"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-align" value=""></button>
+            <button class="ql-align" value="center"></button>
+            <button class="ql-align" value="right"></button>
+          </span>
+          <span class="ql-formats">
+            <select class="ql-color"></select>
+            <select class="ql-background"></select>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-script" value="sub"></button>
+            <button class="ql-script" value="super"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-blockquote"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-list" value="ordered"></button>
+            <button class="ql-list" value="bullet"></button>
+          </span>
+          <span class="ql-formats">
+            <button class="ql-link"></button>
+            <button id="qlImage" class="ql-image" style="display: none;"></button>
+            <button on-tap="_onImage">
+              <iron-icon icon="nuxeo:picture"></iron-icon>
```

Comment:
Why are we making this button click on the hidden #qlImage button? Is it so just we can customize the icon?

Reply:
Adding this `<button class="ql-image">` here is what lets Quill know where the put the default insert image action. Since we want to use a different icon for this action, we need two buttons:
- This invisible one, that Quill will use to inject the logic to trigger the handling of the image upload.
- The next one, which has our own icon and when clicked it just triggers the click on the previous one.
Using the `hidden` attribute instead of the `display: none;` style doesn't hide the first button.
I'll add a short comment here.